contracts/src/v0.8/ccip/pools/GHO/UpgradeableTokenPool.sol

```diff
@@ -102,23 +102,30 @@ abstract contract UpgradeableTokenPool is IPoolV1, Ownable2StepMsgSender {
   address internal immutable i_rmnProxy;
   /// @dev The immutable flag that indicates if the pool is access-controlled.
   bool internal immutable i_allowlistEnabled;
-  /// @dev A set of addresses allowed to trigger lockOrBurn as original senders.
-  /// Only takes effect if i_allowlistEnabled is true.
-  /// This can be used to ensure only token-issuer specified addresses can move tokens.
-  EnumerableSet.AddressSet internal s_allowlist;
-  /// @dev The address of the router
-  IRouter internal s_router;
-  /// @dev A set of allowed chain selectors. We want the allowlist to be enumerable to
-  /// be able to quickly determine (without parsing logs) who can access the pool.
-  /// @dev The chain selectors are in uint256 format because of the EnumerableSet implementation.
-  EnumerableSet.UintSet internal s_remoteChainSelectors;
-  mapping(uint64 remoteChainSelector => RemoteChainConfig) internal s_remoteChainConfigs;
-  /// @notice A mapping of hashed pool addresses to their unhashed form. This is used to be able to find the actually
-  /// configured pools and not just their hashed versions.
-  mapping(bytes32 poolAddressHash => bytes poolAddress) internal s_remotePoolAddresses;
-  /// @notice The address of the rate limiter admin.
-  /// @dev Can be address(0) if none is configured.
-  address internal s_rateLimitAdmin;
+
+  /// @custom:storage-location erc7201:aave-ccip.storage.UpgradeableTokenPool
+  struct UpgradeableTokenPoolStorage {
```

Comment:
are not all of this part of TokenPool storage layout? so we don't need this

Reply:
this is the storage for this contract tho, i'm not sure i follow 
(we don't inherit from `TokenPool` in `UpgradeableTokenPool`, we copy paste and add modifications on top)